DistFiles/aboutBox.htm

```diff
@@ -39,7 +39,11 @@ <h1>Thanks</h1>
         <a href="https://www.unitedbiblesocieties.org/">United Bible Societies</a> and <a href="http://www.everytribeeverynation.org/">Every Tribe Every Nation</a>,
         for their collaboration in producing Paratext and the ParatextData library.
     </p>
-
+    <p>
+        <a href="https://www.faithcomesbyhearing.com/">Faith Comes by Hearing</a> (FCBH), for their partnership in making audio Scriptures widely, for putting together a
```

Comment:
Did you mean something like "widely available"?

Reply:
Yes, something very much like that!